database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -209,23 +211,11 @@ private Schema inferSchema(Class<? extends Driver> driverClass) {
 
   @Override
   public void transform(StructuredRecord input, Emitter<KeyValue<DBRecord, NullWritable>> emitter) {
-    // Create StructuredRecord that only has the columns in this.columns
-    List<Schema.Field> outputFields = new ArrayList<>();
-    for (Schema.Field field : input.getSchema().getFields()) {
-      Preconditions.checkArgument(columns.contains(field.getName()), "Input field '%s' is not found in columns",
-                                  field.getName());
-      outputFields.add(field);
-    }
-    StructuredRecord.Builder output = StructuredRecord.builder(outputSchema);
```

Comment:
since outputSchema is no longer used here, it no longer needs to be set in the initialize() call either, and can probably be removed as a variable.

Reply:
`outputSchema` instance variable removed. Changed to use local variables, did not remove them at all in order to maintain readability.